### ISSUE TITLE
[BITV]: No mouse pointer at search and notification icons

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#42959]
![image](https://github.com/nextcloud/server/assets/42591237/d06ca3a5-3d13-4e36-89ed-bda549aec7ba)
When pointing tat these with a mouse, the mouse is not shown as a pointer which is a hard requirement afaik

**Comment by szaimen** [comment]
cc @JuliaKirschenheuter FYI

**Comment by emoral435** [comment]
Seems like this is only on the contacts menu. GIF:
![firefox_N5dzbiLu1o](https://github.com/nextcloud/server/assets/110193237/1810a43d-14aa-4197-a226-65659e34623f)

Does it still not hover for the unified search for you? I will go ahead and fix up the contacts menu for sure though 👍

**Comment by susnux** [comment]
> Does it still not hover for the unified search for you? I will go ahead and fix up the contacts menu for sure though 👍

Unified search would be covered by https://github.com/nextcloud/server/pull/42978

**Comment by szaimen** [comment]
dependency update in stable28 seems to be missing....

**Comment by emoral435** [comment]
Missing a dependency update to bump NcVue to 8.50.0 👍 will work on that rn!

**Comment by szaimen** [comment]
dependency update for stable28 seems to be missing

**Comment by emoral435** [comment]
Waiting for 8.5.0 (for fix), and 8.5.1 (for styling fixes) - completely my fault, should be fixed soon

**Comment by susnux** [comment]
@emoral435 stable28 is now using 8.5.1, so can be closed?
